src/components/FeedbackSurvey.tsx

```diff
@@ -25,6 +25,9 @@ type FeedbackSurveyProps = {
 
     /** Styles for the option row element */
     optionRowStyles?: StyleProp<ViewStyle>;
+
+    /** Optional text to render over the submit button */
+    bottomText?: React.ReactNode;
```

Comment:
NAB, but generally we've seen using footer.

```suggestion
   footerText?: React.ReactNode;
```